renderer/main.js

```diff
@@ -109,7 +109,7 @@ function onState (err, _state) {
 
   // Log uncaught JS errors
   window.addEventListener('error',
-    (e) => telemetry.logUncaughtError('window', e.error), true)
+    (e) => telemetry.logUncaughtError('window', e.error || e.target), true)
```

Comment:
Okay so this seems to happen when the video element does not know how to interpret the file. For example changing the extension of some garbage file to `.avi` will cause this. The `mediaError` function will still be called with an `undefined` error though, if this is okay I can remove `e.target`?